Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -866,25 +860,25 @@ def rasterize_command():  # pragma: no cover
     rasterize_type = RasterizeType(demisto.args().get('type', 'png').lower())
     wait_time = int(demisto.args().get('wait_time', 0))
     navigation_timeout = int(demisto.args().get('max_page_load_time', DEFAULT_PAGE_LOAD_TIME))
-    file_name = demisto.args().get('file_name', 'url')
+    file_names = argToList(demisto.args().get('file_name', 'url'))
     include_url = argToBoolean(demisto.args().get('include_url', False))
 
     file_extension = "png"
     if rasterize_type == RasterizeType.PDF or str(rasterize_type).lower == RasterizeType.PDF.value:
         file_extension = "pdf"
-    file_name = f'{file_name}.{file_extension}'  # type: ignore
 
     rasterize_output = perform_rasterize(path=url, rasterize_type=rasterize_type, wait_time=wait_time,
                                          navigation_timeout=navigation_timeout, include_url=include_url,
                                          full_screen=full_screen)
     demisto.debug(f"rasterize_command response, {rasterize_type=}, {len(rasterize_output)=}")
-    for current_rasterize_output in rasterize_output:
+    for current_rasterize_output, file_name in zip(rasterize_output, file_names):
         # demisto.debug(f"rasterize_command response, {current_rasterize_output=}")
+        file_name = f'{file_name}.{file_extension}'  # type: ignore
```

Comment:
For better readibility
```suggestion
    for current_rasterize_output, current_file_name in zip(rasterize_output, file_names):
        # demisto.debug(f"rasterize_command response, {current_rasterize_output=}")
        file_name = f'{current_file_name}.{file_extension}'  # type: ignore
```